graylog2-server/src/test/java/org/graylog/testing/containermatrix/annotations/ContainerMatrixTestsConfiguration.java

```diff
@@ -66,7 +66,7 @@
      * matrix rule
      * If no version is explicitly specified, then {@link MongodbServer#DEFAULT_VERSION will be used by the tests}
      */
-    MongodbServer[] mongoVersions() default {MongodbServer.MONGO5};
+    MongodbServer[] mongoVersions() default {MongodbServer.MONGO7};
```

Comment:
I'm not sure if this PR is supposed to be merged or if it was just a test.

If we want to merge it, we should default to the oldest supported MongoDB version to make sure everything works with that version.